packages/frontend/src/components/wallet/Wallet.js

```diff
@@ -39,6 +39,14 @@ const StyledContainer = styled(Container)`
             margin-top: -15px;
         }
     }
+    
+    .coingecko {
+        color: #B4B4B4;
+        line-height: 150%;
+        align-self: end;
+        padding: 10px 20px;
```

Comment:
nit: can we please use margin instead of padding for spacing around the div here, and fix alignment on smaller screens. Something like this should be fine:

```
margin: 20px;
@media (max-width: 991px) {
    margin: -25px 0 25px 0;
}
```